### PR TITLE
OCM-13068 | fix: Add required flags to oprole creation for cluster manual mode

### PR DESCRIPTION
**Pull request by hunterkepley**

### CODE REVIEW

File: cmd/create/cluster/cmd.go [hunk 1]
```diff
@@ -3465,6 +3465,13 @@ func run(cmd *cobra.Command, _ []string) {
 			if !output.HasFlag() || r.Reporter.IsTerminal() {
 				r.Reporter.Infof("Preparing to create operator roles.")
 			}
+			// Set flags for HCP shared VPC if needed
+			if vpcEndpointRoleArn != "" && route53RoleArn != "" && isHostedCP {
+				operatorroles.Cmd.Flags().Set(route53RoleArnFlag, route53RoleArn)
+				operatorroles.Cmd.Flags().Set(vpcEndpointRoleArnFlag, vpcEndpointRoleArn)
+				operatorroles.Cmd.Flags().Set(operatorroles.HostedCpFlag, helper.True)
```
Comment (gdbranco) [inline cmd/create/cluster/cmd.go:3472]:
```suggestion
				operatorroles.Cmd.Flags().Set(operatorroles.HostedCpFlag, isHostedCP)
```
Wouldn't that work?

Comment (hunterkepley) [reply cmd/create/cluster/cmd.go:3472]: should be true no matter what here but would definitely be cleaner

Comment (hunterkepley) [reply cmd/create/cluster/cmd.go:3472]: oh, `.Set` requires a string

Comment (davidleerh) [reply cmd/create/cluster/cmd.go:3472]: how about using "strconv.FormatBool(isHostedCP)"?

**Pull request merged**